src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -35,12 +43,12 @@ internal class DaprClientGrpc : DaprClient
         /// <summary>
         /// Initializes a new instance of the <see cref="DaprClientGrpc"/> class.
         /// </summary>
-        /// <param name="channel">gRPC channel to create gRPC clients.</param>
+        /// <param name="inner">Autogenerated.Dapr.DaprClient.</param>
         /// <param name="jsonSerializerOptions">Json serialization options.</param>
-        internal DaprClientGrpc(GrpcChannel channel, JsonSerializerOptions jsonSerializerOptions = null)
+        internal DaprClientGrpc(Autogenerated.Dapr.DaprClient inner, JsonSerializerOptions jsonSerializerOptions)
```

Comment:
It might be beneficial to just delete the docs. There's no real value for doing that for an internal method (unless you have something interesting to say).